2022/schedule.md

```diff
@@ -1 +1,30 @@
-# Schedule (Coming Soon!)
+# Schedule
+
+| Time (EST) | Plan | Speaker | Chair |
+| :--- | :--- | :--- | :--- |
+| 9:00AM - 9:30AM | [Introduction and State of SciML](/2022/talks#intro) | Chris Rackauckas | N/A |
+| 9:30AM - 9:50AM | [SciML for Climate: Integrating ClimaCore.jl with OrdinaryDiffEq](/2022/talks#climacore) | | Chris |
+| 9:50AM - 10:10AM | [Understanding heterogeneous health outcomes with SciML and Pumas](/2022/talks#pumas) | | Chris |
+| 10:10AM - 10:30AM | [Neural ODEs in Hydrology](/2022/talks#hydrology) | | Chris |
+| 10:30AM - 10:50AM | [Modeling Plasma Physics with NeuralPDE.jl](/2022/talks#neuralplasma) | | Chris |
+| 10:50AM - 11:00AM | Break | | | |
+| 11:00AM - 11:40AM | [Keynote](/2022/talks#keynote) | | Sam |
+| 11:40AM - 12:00PM | [Internals of ModelingToolkit](/2022/talks#mtkinternals) | | Sam |
+| 12:20PM - 12:40PM | [Symbolic arrays: Past, Present and Future](/2022/talks#symbolicarrays) | | Sam |
+| 12:40PM - 1:00PM | [Structural Identifiability Tools in Julia: A Tutorial](/2022/talks#structuralid) |  | Sam |
+| 1:00PM - 1:20PM | [Parallel Extrapolation Methods for Differential Equations](/2022/talks#parallel) | | Sam |
+| 1:20PM - 1:30PM | Break  | | | 
+| 1:30PM - 1:50PM | [Sponsor Talk](/2022/talks#sponsor) | | Ranjan |
+| 1:50PM - 2:10PM	| [Learning measured bifurcation diagrams with UDEs](/2022/talks#udesbif) |  | Ranjan |
```

Comment:
I see no reason to mention session chairs in the announcement? Maybe let's just figure that out internally.

Reply:
It is not unusual and I think it helps, but we can remove them if preferred. I see I was lazy and didn't add the last names.

Reply:
Okay sure. I haven't seen it before so I thought I'd ask. :-)